### PR TITLE
add to v2handler

### PR DESCRIPTION
**Pull request by jessica-christie**
This is needed for hydra tests.

### CODE REVIEW

File: upload/aws_test.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func TestSign(t *testing.T) {
 	resp, err := req.Sign("a", "b")
 	assert.Nil(err)
 	assert.NotEmpty(resp.Date)
-	assert.Contains(resp.Authorization, "AWS4-HMAC-SHA256 Credential=a/20180223/us-east-1/s3/aws4_request, SignedHeaders=host;test-header;x-amz-content-sha256;x-amz-date")
+	assert.Contains(resp.Authorization, "AWS4-HMAC-SHA256 Credential=a/20180312/us-east-1/s3/aws4_request, SignedHeaders=host;test-header;x-amz-content-sha256;x-amz-date")
```
Comment (batmany13) [inline upload/aws_test.go]: will this change for each test run?  Should we have it just look for everything except the date?

File: test_server/server.go [hunk 2]
```diff
@@ -309,6 +309,7 @@ func (s *TestServer) handleV2(w http.ResponseWriter, r *http.Request) {
 		route := "/" + SYNQ_ROUTE
 		switch r.URL.Path {
 		case route + "/videos/" + V2_VIDEO_ID,
+			route + "/videos/" + V2_VIDEO_ID2,
```
Comment (batmany13) [inline test_server/server.go]: I believe some tests expect VIDEO_ID2 to return error, so if this is added, we may need to fix other tests.  Just a FYI

Comment (jessica-christie) [reply test_server/server.go]: I checked aerico, ifrit, pixie, unicorn and they are not using it. I'll check which other repos are using synq-golang.

File: upload/aws_test.go (hunk 1 above)
Comment (jessica-christie) [reply upload/aws_test.go]: I think so? I'll update it.

File: test_server/server.go (hunk 2 above)
Comment (batmany13) [reply test_server/server.go]: ok, we can just fix it as needed

**Pull request merged**